src/main/scala/shell/xilinx/VCU118NewShell.scala

```diff
@@ -460,27 +463,38 @@ abstract class VCU118ShellBasicOverlays()(implicit p: Parameters) extends UltraS
 }
 
 case object VCU118ShellPMOD extends Field[String]("JTAG")
+case object VCU118ShellPMOD2 extends Field[String]("JTAG")
 
 class WithVCU118ShellPMOD(device: String) extends Config((site, here, up) => {
   case VCU118ShellPMOD => device
 })
 
+// Change JTAG pinouts to VCU118 J53
+// Due to the level shifter is from 1.2V to 3.3V, the frequency of JTAG should be slow down to 1Mhz
```

Comment:
This is the reason we are using the FMC instead of the male PMOD

Reply:
Sure, I believe this is key issue to use FMC in stead of J53. But there should be a FMC extension board to pop out the JTAG pins. In my thought, it would be useful if someone need JTAG and SD card at the same time but haven't FMC extension board.
Since SD card can be used for booting Linux and run test cases like SPEC2006 or others, JTAG can be used to debug or monitor CPU status. That's why I create this JTAG port after having some discussions with software teams.
BTW, could you please suggest me the FMC extension board if we need to raise the JTAG throughput? I have some study about it, but I think it should be better to align the model for us. Thanks a lot.

Reply:
Sure, makes sense. https://www.xilinx.com/products/boards-and-kits/hw-fmc-xm105-g.html That is the debug card we usually use